java/src/jmri/jmrit/picker/PickPanel.java

```diff
@@ -64,8 +65,14 @@ private JPanel makeAddToTablePanel() {
         _userNametext = new JTextField();
 
         ActionListener cancelListener = new ActionListener() {
-            public void actionPerformed(ActionEvent e) { cancelPressed(e); }
+            public void actionPerformed(ActionEvent a) {
+                _pickTables.setVisible(false);
```

Comment:
Cancel button doesn't function on Picker Pane, as it is invoked from 3 levels up in LogixAddEdit. Method is not public and in another package. Hide button on Picker Panel? In that case I can't work out how to set cancel.setVisible(false).